ddcommon/src/connector/conn_stream.rs

```diff
@@ -26,16 +26,22 @@ pub enum ConnStream {
         #[pin]
         transport: tokio::net::UnixStream,
     },
+
+    #[cfg(windows)]
+    NamedPipe {
+        #[pin]
+        transport: tokio::net::windows::named_pipe::NamedPipeClient,
+    },
 }
 
 pub type ConnStreamError = Box<dyn std::error::Error + Send + Sync>;
 
 use hyper::{client::HttpConnector, service::Service};
 impl ConnStream {
-    pub async fn from_uds_uri(uri: hyper::Uri) -> Result<ConnStream, ConnStreamError> {
+    pub async fn from_uds_uri(_uri: hyper::Uri) -> Result<ConnStream, ConnStreamError> {
```

Comment:
Did you put `_uri` here because of unused warnings when compiling on `#[cfg(not(unix))`?

If yes, I'd rather put something like

```rust
#[cfg(not(unix))]
{
    let _ = uri;
    Err(super::errors::Error::UnixSocketUnsupported.into())
}
```

Because when I see underscored variables in function signatures, I assume the value will. be unused which is not always the case

Reply:
yeah exactly. Sorry for that, I'm a beginner in Rust.
Thanks, I'll fix it

Reply:
done in https://github.com/DataDog/libdatadog/pull/43/commits/c9ee643c12d20d6c56852cf9d59f1d987f7f867d